src/components/search-bar/SearchBar.tsx

```diff
@@ -0,0 +1,97 @@
+import React, { useState } from "react";
+
+import * as Styled from "./SearchBar.styled";
+import { useStore } from "../../stores/ZustandStore";
+
+import ArrowDown from "../../../public/assets/arrow-down.svg";
+import SearchIcon from "../../../public/assets/magnifying-glass.svg";
+
+interface SearchBarProps {
+  pageTitle: "Timesheets" | "Team members";
+  count: number;
+  clearFilters: Function;
+  isVisible: boolean;
+  searchInput: string;
+  setSearchInput: Function;
+}
+
+function SearchBar({
+  pageTitle,
+  count,
+  clearFilters,
+  isVisible,
+  setSearchInput,
+  searchInput,
+}: SearchBarProps) {
+  const [isListVisible, setIsListVisible] = useState(false);
+
+  const handleClick = () => setIsListVisible(!isListVisible);
+
+  const timeEntries = useStore((state) => state.timeEntries);
+  const setTimeEntries = useStore((state) => state.setTimeEntries);
+
+  const entryFilter = (event) => {
```

Comment:
```suggestion
  const filterTimeEntries = (event) => {
```

I think this would be more descriptive